sdks/python/setup.py

```diff
@@ -334,9 +334,11 @@ def get_portability_package_data():
             # https://github.com/jupyter/jupyter_client/issues/637
             'jupyter-client>=6.1.11,!=6.1.13,<8.1.1',
             'timeloop>=1.0.2,<2',
+            'nbformat>=5.0.5,<6',
```

Comment:
if we already include these libraries in `interactive` - we shouldn't need them in `interactive_test` .